composer.json

```diff
@@ -11,7 +11,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.2",
-        "symfony/event-dispatcher": "^2.0 || ^3.0"
+        "symfony/event-dispatcher": "^2.3 || ^3.0"
```

Comment:
There likely wasn't any reason to bump to 2.3, as the existing code should still have worked with 2.0. I just happened to miss this when merging the PR and tagging 1.1.0.

If the library doesn't actually depend on newer functionality and BC happens to be preserved, I'm in favor of keeping existing requirements as-is.